helm/teleport-operator/values.yaml

```diff
@@ -66,3 +66,4 @@ affinity:
 
 tbot:
   enabled: false
```

Comment:
couldn't we use `tbot.enabled` to decide if we create the tbot resources instead of introducing `tbot.deployment`?

Reply:
I am not sure because its also used for the `--tbot` flag in `teleport-operator` deployment

Reply:
It looks like when enabled, it creates some configuration for `tbot`, so I guess it makes sense that either both are enabled or both disabled at the same time? https://github.com/giantswarm/teleport-operator/pull/143/files